### PR TITLE
Improve template sysctl to support other types of values

### PR DESCRIPTION
**Pull request by yuumasato**
#### Description:

- Extend sysctl templates to support other data types
- Update docs in sysctl_value CSV files 

#### Rationale:
- Most of the sysctl values are integers, but some can be strings.

### CODE REVIEW

File: shared/templates/create_sysctl.py [hunk 1]
```diff
@@ -33,8 +33,19 @@ def get_files_for_id(self, var_id):
...
-        sysctl_var, sysctl_val = serviceinfo
+        # items can be in format
+        # <sysctl_parameter_name, value> or
+        # <sysctl_parameter_name, value, type>
+        try:
+            sysctl_var, sysctl_val = serviceinfo
+            # Default data type for sysctl is int
+            data_type = "int"
+        except ValueError as e:
+            if str(e) == "too many values to unpack (expected 2)":
```
Comment (yuumasato) [inline shared/templates/create_sysctl.py]: Maybe I'm being overly precautious here.

Comment (matejak) [reply shared/templates/create_sysctl.py]: I would go for `if len(serviceinfo) == 3`, the examination of the exception error message may change, and four values would have the same message as three.

File: shared/templates/create_sysctl.py [hunk 2]
```diff
@@ -117,4 +130,4 @@ def generate(self, target, serviceinfo):
 
     def csv_format(self):
         return("CSV should contains lines of the format: " +
-               "sysctlvariable,sysctlvalue")
+               "sysctlvariable,sysctlvalue,<datatype>")
```
Comment (matejak) [inline shared/templates/create_sysctl.py]: Let's use `[,<datetype>]` in the help message - we already have this marker of optional options elsewhere.

File: shared/templates/create_sysctl.py (hunk 1 above)
Comment (yuumasato) [reply shared/templates/create_sysctl.py]: Fair enough, updated per suggestion.

Comment (yuumasato) [reply shared/templates/create_sysctl.py]: Also dealt with exception case, when CSV line is out of the norm

File: shared/templates/create_sysctl.py (hunk 2 above)
Comment (yuumasato) [reply shared/templates/create_sysctl.py]: Updated.

**Comment by matejak** [comment]
I am OK with the PR now, but the CI seems to start finding issues.

**Comment by yuumasato** [comment]
A test CSV line was pushed by mistake, it should be fixed now.

**Comment by matejak** [comment]
Thank you for the PR, merging!

**Pull request merged**